### PR TITLE
feat(a380x): OANS / BTV Advanced Guide

### PR DESCRIPTION
**Pull request by flogross89**
### Location

Discord username (if different from GitHub): floridude

### CODE REVIEW

File: docs/pilots-corner/a380x/a380x-advanced-guides/oans-btv.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+The OANS airport map is active in ROSE NAV, ARC and PLAN EFIS mode. In PLAN mode, any airport of the database can be displayed. In all modes, the center of the map can be moved using the mouse (click and drag in the instrument).
+
+If you want to center the map on the aircraft again, double click on the map to open the context menu and select "CENTER ON ACFT".
+
+![Center on ACFT](../assets/advanced-guides/oans-btv/oans-center-on-acft.png)
+
+## Approaching Runway Advisory
+
+If the aircraft is on ground and ground speed is below 40 knots, OANS continuously checks the aircraft protection volume in front of the aircraft (as widge as the wingspan, foresight of 7 seconds) for runways. When detecting a runway inside this volume, an advisory is triggered on the PFD and ND. The advisory disappears, if the above conditions aren't met anymore, or the aircraft entered the runway. There is no aural warning (as RAAS).
+![Runway ahead](../assets/advanced-guides/oans-btv/oans-runway-ahead.png)
```
Comment (frankkopp) [inline docs/pilots-corner/a380x/a380x-advanced-guides/oans-btv.md:47]:
```suggestion

![Runway ahead](../assets/advanced-guides/oans-btv/oans-runway-ahead.png)
```

File: docs/pilots-corner/a380x/a380x-advanced-guides/oans-btv.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+The OANS airport map is active in ROSE NAV, ARC and PLAN EFIS mode. In PLAN mode, any airport of the database can be displayed. In all modes, the center of the map can be moved using the mouse (click and drag in the instrument).
+
+If you want to center the map on the aircraft again, double click on the map to open the context menu and select "CENTER ON ACFT".
+
+![Center on ACFT](../assets/advanced-guides/oans-btv/oans-center-on-acft.png)
+
+## Approaching Runway Advisory
+
+If the aircraft is on ground and ground speed is below 40 knots, OANS continuously checks the aircraft protection volume in front of the aircraft (as widge as the wingspan, foresight of 7 seconds) for runways. When detecting a runway inside this volume, an advisory is triggered on the PFD and ND. The advisory disappears, if the above conditions aren't met anymore, or the aircraft entered the runway. There is no aural warning (as RAAS).
```
Comment (frankkopp) [inline docs/pilots-corner/a380x/a380x-advanced-guides/oans-btv.md]: I would define volume or use another term like "area". 
At least for me volume seems a strange term here and harder to understand. But I'm no native speaker so I might be wrong.

**Review by frankkopp — CHANGES_REQUESTED** [review]
Thanks for this guide!
Some remarks...

For images we have some standard sizes. 
Full width is 826px. Using this you avoid browser resizing which introduces some blur in most cases. 

Or you could use width 1024px as clicking on the images pops them out and shows them larger. Good for reading details. 

Table with image sizes is pinned in the documentation Discord channel. 

And all images need to optimized to <100kB - I can do this for you at the end. But there are many free png optimizers out there and is an easy step. 

If you also could keep lines <120 char it would be nice. Same as with code long lines are harder to read when using an IDE when autowrap is not turned on (which at least I never have). 
MD does not care if you add a line break in long lines. Will not be visible in the output.

File: docs/pilots-corner/a380x/a380x-advanced-guides/oans-btv.md (hunk 2 above)
Comment (flogross89) [reply docs/pilots-corner/a380x/a380x-advanced-guides/oans-btv.md]: Volume is the term which Airbus uses but I'll use area from now. Also more suitable to less technical people

**Comment by flogross89** [comment]
> Thanks for this guide! Some remarks...
> 
> For images we have some standard sizes. Full width is 826px. Using this you avoid browser resizing which introduces some blur in most cases.
> 
> Or you could use width 1024px as clicking on the images pops them out and shows them larger. Good for reading details.
> 
> Table with image sizes is pinned in the documentation Discord channel.
> 
> And all images need to optimized to <100kB - I can do this for you at the end. But there are many free png optimizers out there and is an easy step.
> 
> If you also could keep lines <120 char it would be nice. Same as with code long lines are harder to read when using an IDE when autowrap is not turned on (which at least I never have). MD does not care if you add a line break in long lines. Will not be visible in the output.

Thanks for the review! I just commited a new version. 
- I didn't have any image over 826px, but I resized all my images to match your table and compressed them
- I also introduced some line breaks now, amongst other small things

**Review by frankkopp — APPROVED** [review]

**Comment by frankkopp** [comment]
I will merge this now - any changes we can also make directly in the target branch. 
Brings things together more quickly.

**Pull request merged**